src/election.ts

```diff
@@ -88,13 +88,23 @@ export type Vote = CandidateVote | YesNoVote
 export type OptionalVote = Optional<Vote>
 export type VotesDict = Dictionary<Vote>
 
+export enum BallotType {
+  Standard = 0,
+  Absentee = 1,
+  Provisional = 2,
+}
+
+// Update this whenever a new BallotType case is added.
+export const BallotTypeMaximumValue = BallotType.Provisional
```

Comment:
Maybe leave room in the encoding for a few more, just so we can have encoding format forward compatibility if we just add a ballot type or two?

Reply:
Hah, we typed more or less the same comment at the same time. Will do.